### ISSUE TITLE
Cavity Flow Exercise

### ISSUE THREAD

**Issue by nicoguaro** [gforsyth/numba_tutorial_scipy2017#7]
There is another issue in the ``05.Cavity.Flow.Exercises.ipynb`` notebook

````python
with open('numpy_ans.pickle', 'rb') as f:
    u, v, p = pickle.load(f)
````

needs to be

````python
with open('../numpy_ans.pickle', 'rb') as f:
    u, v, p = pickle.load(f)
````

**Comment by gforsyth** [comment]
Thanks @nicoguaro !  I need to do a once over on the tutorial to update it for recent Numba releases.  In the interim, a pull request fixing the small errors would be most welcome!

**Comment by nicoguaro** [comment]
@gforsyth, I just found that typo. I already created the pull request.

**Comment by gforsyth** [comment]
Resolved in #9
